### DISCUSSION TITLE
Doo-re Backend Infra 문서화

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#316] [1. 개발 관련 글 모음]
[Infra - Doo-re Infra(1) - 도커란 무엇인가?](https://02ggang9.github.io/infra/DooreInfra2/)

[Infra - Doo-re Infra(2) - Gradle과 JAR](https://02ggang9.github.io/infra/DooreInfra3/)

[Infra - Doo-re Infra(3) - Infra Code 작성 및 로컬 환경에서 테스트](https://02ggang9.github.io/infra/DooreInfra4/)

[Infra - Doo-re Infra(4) - AWS EC2 인스턴스 생성](https://02ggang9.github.io/infra/DooreInfra5/)

[Infra - Doo-re Infra(5) - EC2 환경 설정과 도커 띄우기](https://02ggang9.github.io/infra/DooreInfra6/)

[Infra - Doo-re Infra(6) - Git Action을 위한 권한 설정 Configure AWS Credntials](https://02ggang9.github.io/infra/DooreInfra7/)

[Infra - Doo-re Infra(7) - ECR 사용을 위한 환경 설정](https://02ggang9.github.io/infra/DooreInfra8/)

[Infra - Doo-re Infra(8) - S3 버킷 생성 방법과 Slack 연동 방법](https://02ggang9.github.io/infra/DooreInfra9/)

[Infra - Doo-re Infra(9) - GITHUB ACTION 코드 살펴보기](https://02ggang9.github.io/infra/DooreInfra10/)

[yml 분리 이유와 caused by: org.yaml.snakeyaml.constructor.duplicatekeyexception at safeconstructor.java 에러 해결](https://02ggang9.github.io/trouble/DuplicateKeyException/)

[java.lang.IllegalStateException at DefaultCacheAwareContextLoaderDelegate.java 에러 해결](https://02ggang9.github.io/trouble/DefaultCacheAware/)

[docker :: invalid reference format: repository name must be 에러 해결](https://02ggang9.github.io/trouble/DockerInvalidFormat/)

[org.springframework.boot.loader.JarLauncher cannot be found 에러 해결](https://02ggang9.github.io/trouble/JarLauncher/)

[It looks like you might be trying to authenticate with OIDC. Did you mean to set the id-token permission? 에러 해결](https://02ggang9.github.io/trouble/ConfigureAwsCredentials/)

**Comment by shkisme** [comment]
헐 랭
